policyengine/utils/data/caching_google_storage_client.py

```diff
@@ -39,28 +61,30 @@ def download(self, bucket: str, key: str, target: Path):
 
     # If the crc has changed from what we downloaded last time download it again.
     # then update the CRC to whatever we actually downloaded.
-    def sync(self, bucket: str, key: str) -> None:
+    def sync(self, bucket: str, key: str, version: str | None = None) -> None:
         """
         Cache the resource if the CRC has changed.
         """
-        logger.info(f"Syncing {bucket}, {key} to cache")
-        datakey = f"{bucket}.{key}.data"
-        crckey = f"{bucket}.{key}.crc"
+        logger.info(f"Syncing {bucket}, {key}, {version} to cache")
+        datakey = f"{bucket}.{key}.{version}.data"
+        crckey = f"{bucket}.{key}.{version}.crc"
 
-        crc = self.client.crc32c(bucket, key)
+        crc = self.client.crc32c(bucket, key, version=version)
         if crc is None:
             raise Exception(f"Unable to find {key} in bucket {bucket}")
 
         prev_crc = self.cache.get(crckey, default=None)
-        logger.debug(f"Previous crc for {bucket}, {key} was {prev_crc}")
+        logger.info(f"Previous crc for {bucket}, {key} was {prev_crc}")
```

Comment:
__suggestion, non-blocking:__ Might consider reverting to previous
```suggestion
        logger.debug(f"Previous crc for {bucket}, {key} was {prev_crc}")
```